packages/fx-core/resource/yaml.schema.json

```diff
@@ -1128,6 +1128,10 @@
             "target": {
               "type": "string",
               "description": "the target file"
+            },
+            "content": {
+              "type": "object",
+              "description": "the json content to be created or updated"
```

Comment:
Should we tell users that the it is incremental update in case users always put the whole json content here?

Reply:
Thanks for the advice, done.